.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,16 +23,23 @@
 - _BE GENEROUS to your reviewer and your future self!_
 
 ### Checks to be considered:
+If any of these don't apply to your situation, please write N/A (not applicable) next to them and tick them.
+- [ ] Descriptive pull request title
+- [ ] Completed all the questions above
 - [ ] Performed a *self-review* of the code
 - [ ] Additional *automated test* coverage
-  - Unit tests
-  - Integration tests
-- [ ] *3 green runs* in dev environment
+  - [ ] Unit tests
+  - [ ] Integration tests
+  - [ ] System tests (if appropriate)
+  - [ ] Manual testing (e.g. using Postman) has been completed
+- [ ] *3 green runs of [system-tests](https://github.com/SecclTech/system-tests)* in dev environment
```

Comment:
Do we need to duplicate for system-tests-core?

Reply:
yes absolutely